scripts/scene_elements/item.py

```diff
@@ -0,0 +1,51 @@
+import dataclasses
+from typing import Dict, List
+
+import snecs
+
+from scripts.core.data import Data
+from scripts.core.effect import load_effect
+
+
+@dataclasses.dataclass
+class ItemData:
+    """
+    Schema for the YAML Item data files
+
+    """
+
+    name: str
+    is_signature: bool
+    effects: List[Dict[str, str]]
+
+
+class Item(snecs.RegisteredComponent):
+    """
+    Item component -- WIP
+
+    """
+
+    def __init__(self, name, is_signature, effects):
+        self.name: str = name
+        self.is_signature = is_signature
+        self.effects = effects
+
+
+def create_item(data: Data, name: str):
```

Comment:
Do we only need Data as this is a function?

Reply:
it could be rewritten to just accept an `ItemData` object